tests/unit/literaturesuggest/test_literaturesuggest_form.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+from inspirehep.modules.literaturesuggest.forms import LiteratureForm
+
+
+def test_degree_choices_contains_capitalized_phd():
+    """Ensure that lowercased degree type in the schema ends up with the
+    right capitalization in the form.
+    """
+    literature_form = LiteratureForm()
+
+    assert ('phd', 'PhD') in literature_form.degree_type.choices
```

Comment:
> BTW, this was not even what the user wanted! 

❓ I think those were different issues and indeed this commit was saying addresses https://github.com/inspirehep/inspire-next/issues/2161 when it should not have.

Here there was just a display issue being addressed. Before, the dropdown was capitalizing all the options so a user would see `Phd` in the dropdown and what we wanted is to have `PhD` displayed.

So yeah, either it could be a regression test asserting on the wtforms field, or we can assert during the acceptance test of the form that the value `PhD` appears in the dropdown.